src/tooltip/tooltip.directive.ts

```diff
@@ -2,7 +2,7 @@
 import {
   Directive,
   ElementRef,
-  EventEmitter,
+  EventEmitter, HostBinding,
```

Comment:
please keep import syntax consistent, move `HostBinding` to a new line

Reply:
fixed